apps/server/src/modules/user/service/user.service.spec.ts

```diff
@@ -123,6 +141,30 @@ describe('UserService', () => {
 		});
 	});
 
+	describe('findById', () => {
+		beforeEach(() => {
+			const userDO: UserDO = new UserDO({
+				firstName: 'firstName',
+				lastName: 'lastName',
+				email: 'email',
+				schoolId: 'schoolId',
+				roleIds: ['roleId'],
+				externalId: 'externalUserId',
+			});
+			userDORepo.findById.mockResolvedValue(userDO);
+		});
+
+		it('should provide information about the passed userId', async () => {
+			// Act
+			const result: UserDO = await service.findById('id');
+
+			// Assert
+			expect(result).toBeDefined();
+			expect(result).toBeInstanceOf(UserDO);
+			expect(userDORepo.findById).toHaveBeenCalled();
```

Comment:
this third expectation does not match the description of the testcase. either remove or put into seperate test.

Reply:
done